resources/lang.js

```diff
@@ -129,6 +129,15 @@ class CactbotLanguage {
       EurekaPyros: /(Eureka Pyros|Unknown Zone \(31B\))/,
       EurekaHydatos: /(Eureka Hydatos|Unknown Zone \(33B\))/,
     });
+
+    this.kLanguage = Object.freeze([
```

Comment:
(tiniest of bikesheds, but maybe `kLanguages` plural?)

Reply:
I'll make this adjustment.